subsystem/kubeapi_test.go

```diff
@@ -1729,35 +1731,52 @@ var _ = Describe("[kube-api]cluster installation", func() {
+		}, "30s", "10s").ShouldNot(Equal(""))
 
-		By("Add Day 2 host and approve agent")
-		configureLocalAgentClient(cluster.ID.String())
-		host := setupNewHost(ctx, "hostnameday2", *cluster.ID)
-		key := types.NamespacedName{
-			Namespace: Options.Namespace,
-			Name:      host.ID.String(),
-		}
-		generateApiVipPostStepReply(ctx, host, true)
-		Eventually(func() error {
-			agent := getAgentCRD(ctx, kubeClient, key)
-			agent.Spec.Approved = true
-			return kubeClient.Update(ctx, agent)
-		}, "30s", "10s").Should(BeNil())
+		By("Check ACI Logs URL exists")
+		Eventually(func() string {
+			aci := getAgentClusterInstallCRD(ctx, kubeClient, installkey)
+			return aci.Status.DebugInfo.LogsURL
+		}, "30s", "10s").ShouldNot(Equal(""))
 
-		checkAgentCondition(ctx, host.ID.String(), controllers.InstalledCondition, controllers.InstallationInProgressReason)
-		checkAgentCondition(ctx, host.ID.String(), controllers.ReadyForInstallationCondition, controllers.AgentAlreadyInstallingReason)
-		checkAgentCondition(ctx, host.ID.String(), controllers.SpecSyncedCondition, controllers.SyncedOkReason)
-		checkAgentCondition(ctx, host.ID.String(), controllers.ConnectedCondition, controllers.AgentConnectedReason)
-		checkAgentCondition(ctx, host.ID.String(), controllers.ValidatedCondition, controllers.ValidationsPassingReason)
+		// By("Verify Day 2 Cluster")
+		// checkAgentClusterInstallCondition(ctx, installkey, controllers.ClusterCompletedCondition, controllers.InstalledReason)
+		// cluster = getClusterFromDB(ctx, kubeClient, db, clusterKey, waitForReconcileTimeout)
+		// Expect(*cluster.Kind).Should(Equal(models.ClusterKindAddHostsCluster))
+
+		// By("Verify ClusterDeployment Agents were deleted")
+		// Eventually(func() int {
+		// 	return len(getClusterDeploymentAgents(ctx, kubeClient, clusterKey).Items)
+		// }, "2m", "2s").Should(Equal(0))
+
+		// By("Add Day 2 host and approve agent")
+		// configureLocalAgentClient(cluster.ID.String())
+		// host := setupNewHost(ctx, "hostnameday2", *cluster.ID)
+		// key := types.NamespacedName{
+		// 	Namespace: Options.Namespace,
+		// 	Name:      host.ID.String(),
+		// }
+		// generateApiVipPostStepReply(ctx, host, true)
+		// Eventually(func() error {
+		// 	agent := getAgentCRD(ctx, kubeClient, key)
+		// 	agent.Spec.Approved = true
+		// 	return kubeClient.Update(ctx, agent)
+		// }, "30s", "10s").Should(BeNil())
+
+		// checkAgentCondition(ctx, host.ID.String(), controllers.InstalledCondition, controllers.InstallationInProgressReason)
+		// checkAgentCondition(ctx, host.ID.String(), controllers.ReadyForInstallationCondition, controllers.AgentAlreadyInstallingReason)
+		// checkAgentCondition(ctx, host.ID.String(), controllers.SpecSyncedCondition, controllers.SyncedOkReason)
+		// checkAgentCondition(ctx, host.ID.String(), controllers.ConnectedCondition, controllers.AgentConnectedReason)
+		// checkAgentCondition(ctx, host.ID.String(), controllers.ValidatedCondition, controllers.ValidationsPassingReason)
```

Comment:
I think it would have been better to remove this code in the PR targeting master. We have git to find deleted code if we need this in the future.